### PR TITLE
add rbac sync docs

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/gitops/use-gitops/manage-permissions.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: Manage access for GitOps using labels
```
Comment (SudarshanPurohit) [inline docs/continuous-delivery/gitops/use-gitops/manage-permissions.md:2]: This feature is only for Harness GitOps Applications btw, I think we should update the titles to reflect that?

File: docs/continuous-delivery/gitops/use-gitops/manage-permissions.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+## Implementing label-based RBAC
+
+### Step 1: Define your labeling strategy
+
+Before you start, define a consistent labeling strategy. For example:
+- `harness.io/env-type: [prod|staging|dev]`
+- `harness.io/team: [frontend|backend|data]`
+- `harness.io/criticality: [high|medium|low]`
+
+### Step 2: Update your ApplicationSets
```
Comment (SudarshanPurohit) [inline docs/continuous-delivery/gitops/use-gitops/manage-permissions.md:35]: This was specifically implemented for Applications; For AppSets, the template and data used by the AppSet to create the Applications dynamically needs to have the labels in it (as you've explained below). This is the more complex flow. Maybe focus on Applications first and then explain how to do it for AppSets?

File: docs/continuous-delivery/gitops/use-gitops/manage-permissions.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+### Step 3: Create resource groups in Harness
+
+Resource Groups in Harness allow you to group applications based on their labels. Here's how to create them:
+1. Navigate to **Access Control** > **Resource Groups** in your **Project Settings**.
+1. Click **New Resource Group**.
+1. Name your group and click **Save**.
+1. Select your **Resources** on the left. In this case you want **Gitops** > **Applications**
+1. Select **By Label** instead of **All**.
+1. Click **Add**
+1. Type in your label you chose by your naming convention in step 1.
```
Comment (SudarshanPurohit) [inline docs/continuous-delivery/gitops/use-gitops/manage-permissions.md:73]: Maybe take the example of `harness.io/env-type: [prod|staging|dev]` again here? For a prod application resource group, use the label `harness.io/env-type:prod`, and for the dev resource group, use `harness.io/env-type:dev`?

**Review by SudarshanPurohit — COMMENTED** [review]
Looks good, but could we go step by step, explain the flow for simple applications first before going to AppSets?

**Review by SmritiSatya — DISMISSED** [review]

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**